core/src/main/java/org/apache/carbondata/core/indexstore/ExtendedBlocklet.java

```diff
@@ -168,7 +175,7 @@ public void setColumnSchema(List<ColumnSchema> columnSchema) {
    * @throws IOException
    */
   public void serializeData(DataOutput out, Map<String, Short> uniqueLocation, boolean isCountJob,
-      boolean isExternalPath)
+      boolean isExternalPath, CdcVO cdcVO)
```

Comment:
please add comment for all parameters in javadoc

Reply:
done

Reply:
done